apps/prairielearn/src/pages/administratorCourses/administratorCourses.html.ts

```diff
@@ -350,7 +350,13 @@ function CourseUpdateColumnForm({
       <input type="hidden" name="course_id" value="${course.id}" />
       <input type="hidden" name="column_name" value="${column_name}" />
       <div class="form-group">
-        <input type="text" class="form-control" name="value" value="${course[column_name]}" />
+        <input
+          type="text"
+          class="form-control"
+          name="value"
+          value="${course[column_name]}"
+          aria-label="${column_name}"
```

Comment:
Another case where this could render just a column name. The function that calls this one (`CourseUpdateColumn`) receives a `label` prop, consider passing that down and using it here.

Reply:
Now passing down a `label` prop in [b2ddc80](https://github.com/PrairieLearn/PrairieLearn/pull/10514/commits/b2ddc80c9af2e30a4fa6ea94027150ce2ac8b3ba).